src/browser-runner/platform-electron/sagas/server-command-handlers.js

```diff
@@ -28,6 +28,14 @@ function* createWindow({ meta: client, payload: { winId, url, width, height, sty
     }[style],
   });
 
+  win.on('close', (e) => {
+    if (!win.actuallyClose) {
+      e.preventDefault();
+      return false;
+    }
+    return true;
```

Comment:
I'm unfamiliar with this electron event, care to add a comment detailing why we need to return false/true in addition to calling `preventDefault()`?